lib/utils.js

```diff
@@ -5,6 +5,10 @@ var getRandomValue = (value, offset) => {
   return Math.random()*(value||1) + (offset||0);
 }
 
+var rando = (from, to) => {
+  return Math.floor(getRandomValue(to-from + 1, from));
```

Comment:
you can replace the Math.floor by ~~
so `return ~~getRandomValue(to-from + 1, from)`;